lib/tesla/middleware/retry.ex

```diff
@@ -106,7 +106,7 @@ defmodule Tesla.Middleware.Retry do
     jitter = 1 + 2 * jitter_factor * :rand.uniform() - jitter_factor
 
     # The actual delay is in the range max_sleep * (1 - jitter_factor), max_sleep * (1 + jitter_factor)
-    delay = trunc(max_sleep + jitter)
+    delay = trunc(max_sleep * jitter)
```

Comment:
@teamon What do you think about changing it to:

`delay = trunc(base * factor * jitter) |> min(cap)`

this way we get a hard limit on `cap`, but still have jitter around the target value whenever possible.

It's basically the same code as we have here but with the `min(cap, _)` moved to the end of the calculation

Reply:
If you prefer your suggestion anyway, just let me know and I'll update accordingly

Reply:
Actually, we'd need to still calculate the `max_sleep` first anyway. I'm going to use your suggestion which keeps the code simpler

Reply:
Since the base grows exponentially, you'll quickly end up with cap without jitter:

```
49
94
168
353
841
1747
3625
5000
5000
5000
5000
5000
5000
5000
5000
5000
5000
5000
5000
5000
5000
```

Reply:
I've submitted your suggestion :)

Reply:
Oh wait, gotta update the comment above it :)